pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -224,6 +224,13 @@ abstract class Analytics {
   /// Setting the telemetry status will also send an event to GA
   /// indicating the latest status of the telemetry from [reportingBool]
   Future<void> setTelemetry(bool reportingBool);
+
+  /// Calling this will result in telemetry collection being suppressed for
+  /// the current invocation
```

Comment:
```suggestion
  /// the current invocation.
```

Reply:
it's a nit you don't have to fix here, but really all dartdocs should be complete sentences, ending in periods.

Reply:
Ah good point, going to create a separate issue for this
- https://github.com/dart-lang/tools/issues/126